src/components/Item/Item.component.jsx

```diff
@@ -9,18 +10,45 @@ const Item = props => {
   const itemId = props.id.toString();
   const localToken = props.localToken;
   const over24 = props.over24;
+  let lastEstimate = props.lastEstimate;
+  let latestInterval = props.latestInterval;
+  let numberOfPurchases = props.numberOfPurchases;
+  let nextPurchaseInterval = props.nextPurchaseInterval;
+  let lastPurchaseDate = props.date;
```

Comment:
You can unpack the props as arguments, so you don't have to declare them all unless you want to change the variable names of the props.
```suggestion
 const Item = ({name, id, localToken, over24, lastEstimate, latestInterval, numberOfPurchases, nextPurchaseInterval, date}) => {
```

Reply:
It didn't let me choose the right part to edit, so please don't commit this suggestion and change it manually if you like the change!

Reply:
Thank you for the great suggestions, Jen.

Reply:
@CodeMeKathy please note `itemId` needs to be converted to String. Thank you.